### PR TITLE
Added short descriptions for home page

### LINKED ISSUE #39: Content: 4 short descriptions needed to add in homepage.
## Description
As a Developer,
I need to have 4 descriptions (getting started, ways to contribute, our projects, open source programs),
so that I can place them in homepage following the agreed mockup.

## Mocks
[Agreed mockup](https://anitab-org.zulipchat.com/user_uploads/16723/uvNAUxnyTGnUq1hhHCNiNJFU/AnitaB_OS_Website-1st-Iteration.pdf)

### Update [Required]
- [ ] `getting started` short descriptions
- [ ] `ways to contribute` short descriptions
- [ ] `our projects` short descriptions
- [ ] `open source programs` short descriptions

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
[3] hours

### PR DESCRIPTION
**Pull request by vj-codes**
### Description

Added short descriptions for home page.

Fixes #39 

### Type of Change:
Added a JSON file

- Documentation

### CODE REVIEW

**Comment by vj-codes** [comment]
@annabauza here's my first PR ! please review it .Thanks in advance:)

File: src/home_content1.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
+Getting Started
```
Comment (annabauza) [inline src/home_content1.json:1]: we need to organise this in proper json structure
```
{sections:[{ title:"Getting Started",content:[{par:"The AnitaB.org community is a fusion of rich values and diverse cultures consisting of members all across the seven oceans with a fearless mission of inspiring ,motivating and helping women with their involvement in technical fields and has made a significant impact with regards to contribution of women in tech.The other values of the organization include Accountable, Agile, Curious, Courageous, & Diversity, Equity, Inclusion (DEI), encourage everyone to follow the org’s contributing guidelines and obviously passion for open source."},{par:"Zulip is our main communication platform and you should join us there and first give a short introduction about yourself in the #newcomers stream.Zulip consists of various streams like Documentation,Coding,Design,etc and the one that fascinates you the most is the one you should start contributing in.The #announcements stream will be displaying all the important announcements.Ttag our community members if there's any confusion or ask for proper guidance from them regarding your contributions to the community."}...
```

**Comment by annabauza** [comment]
structure example:
```
{
  "sections": [
    {
      "title": "First section title",
      "content": [
        {
          "par": "The AnitaB.org community is a fusion of rich values and diverse cultures consisting of members all across the seven oceans with a fearless mission of inspiring ,motivating and helping women with their involvement in technical fields and has made a significant impact with regards to contribution of women in tech.The other values of the organization include Accountable, Agile, Curious, Courageous, & Diversity, Equity, Inclusion (DEI), encourage everyone to follow the org’s contributing guidelines and obviously passion for open source."
        },
        {
          "par": "Zulip is our main communication platform and you should join us there and first give a short introduction about yourself in the #newcomers stream.Zulip consists of various streams like Documentation,Coding,Design,etc and the one that fascinates you the most is the one you should start contributing in.The #announcements stream will be displaying all the important announcements.Ttag our community members if there's any confusion or ask for proper guidance from them regarding your contributions to the community."
        }
      ]
    },
    {
      "title": "Next section title",
      "content": [
        {
          "par": "The AnitaB.org community is a fusion of rich values and diverse cultures consisting of members all across the seven oceans with a fearless mission of inspiring ,motivating and helping women with their involvement in technical fields and has made a significant impact with regards to contribution of women in tech.The other values of the organization include Accountable, Agile, Curious, Courageous, & Diversity, Equity, Inclusion (DEI), encourage everyone to follow the org’s contributing guidelines and obviously passion for open source."
        },
        {
          "par": "Zulip is our main communication platform and you should join us there and first give a short introduction about yourself in the #newcomers stream.Zulip consists of various streams like Documentation,Coding,Design,etc and the one that fascinates you the most is the one you should start contributing in.The #announcements stream will be displaying all the important announcements.Ttag our community members if there's any confusion or ask for proper guidance from them regarding your contributions to the community."
        },
        {
          "par": "Zulip is our main communication platform and you should join us there and first give a short introduction about yourself in the #newcomers stream.Zulip consists of various streams like Documentation,Coding,Design,etc and the one that fascinates you the most is the one you should start contributing in.The #announcements stream will be displaying all the important announcements.Ttag our community members if there's any confusion or ask for proper guidance from them regarding your contributions to the community."
        }
      ]
    }
  ]
}
```

**Comment by vj-codes** [comment]
okay thanks @annabauza will do the changed soon.

**Pull request closed without merging**